Makefile

```diff
@@ -84,9 +86,15 @@ else
 
 # Not running in Dapper
 
+Makefile.dapper:
+	@echo Downloading $@
+	@curl -sfL -o $@ https://raw.githubusercontent.com/submariner-io/shipyard/$(BASE_BRANCH)/$@
```

Comment:
```suggestion
	@curl -sfL -O https://raw.githubusercontent.com/submariner-io/shipyard/$(BASE_BRANCH)/$@
```

Reply:
Thanks!
Had to do it manually to avoid "Commit suggestion" commit since this PR has 2 distinct commits (though very simple ones indeed)